nectar/src/ethereum/wallet.rs

```diff
@@ -346,6 +306,12 @@ impl Wallet {
         self.geth_client.get_balance(self.account()).await
     }
 
+    pub(crate) async fn erc20_balance(&self, token_contract: Address) -> anyhow::Result<Erc20> {
```

Comment:
I believe this is useless in the context of a binary crate.

Reply:
True, is an artifact of amending several commits :sweat_smile: